activation/atxdb.go

```diff
@@ -178,23 +189,23 @@ func (db *DB) createTraversalActiveSetCounterFunc(countedAtxs map[string]types.A
 
 	traversalFunc := func(b *types.Block) (stop bool, err error) {
 
-		// don't count ATXs in blocks that are not destined to the prev epoch
-		if b.LayerIndex.GetEpoch(db.LayersPerEpoch) != epoch-1 {
+		// count unique ATXs
+		if b.ActiveSet == nil {
```

Comment:
does this work?  I think unless you use a pointer or explicitly set to nil and array will init as an empty array and not nil. (len(0)/[])

Reply:
this is how it's deserialized from XDR - to nil if doesnt exist